src/main/java/crazypants/enderio/machine/farm/FarmStationContainer.java

```diff
@@ -9,6 +9,18 @@
 
 public class FarmStationContainer extends AbstractMachineContainer {
 
+  private static final int ROW_TOOLS = 19;
+  private static final int ROW_IO = 44;
+
+  private static final int COL_TOOLS = 44;
+  private static final int COL_INPUT = 53;
+  private static final int COL_OUTPUT = 116;
```

Comment:
Could you move the output slots over half a slot width? So that they are centered on where they used to be. I think it looks weird that close to the edge. Just a nit-pick.